### PR TITLE
Savva Dzhumaliev Tasks 51-60

### PR DESCRIPTION
**Pull request by EasileleM**

### CODE REVIEW

File: tasks/58_missed-letters/missed-letters.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export function findLetter(str) {
+  let prevCharCode = str.charCodeAt(0);
+  for (let i = 1; i < str.length; i++) {
+    const currCharCode = str.charCodeAt(i);
+    if (prevCharCode + 1 !== currCharCode) {
+      return String.fromCharCode(prevCharCode + 1);
+    }
+    prevCharCode = currCharCode;
+  }
+  return undefined;
```
Comment (akimovaa) [inline tasks/58_missed-letters/missed-letters.js]: function return undefined by default

**Review by akimovaa — DISMISSED** [review]

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ * abbreviated - converting full name in
```
Comment (vovcyan) [inline tasks/51_abbreviated-name/abbreviated-name.js:2]: Lets put some example here
Data before transforming and after

File: tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+/**
+ * sumAll - returns sum of all numbers in given range
```
Comment (vovcyan) [inline tasks/52_sum-all-numbers-in-range/sum-all-numbers-in-range.js:2]: Lets put some example

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by akimovaa — DISMISSED** [review]

File: tasks/57_pig-latin/pig-latin.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/**
+ * translatePigLatin - translates provided word into Pig Latin
+ * @example
+ * // return "eightway"
+ * translatePigLatin('eight')
+ * @param {String} str
+ * @return {String}
+ */
+export function translatePigLatin(str) {
+  const vowels = 'qeuioa';
```
Comment (savchukm) [inline tasks/57_pig-latin/pig-latin.js]: q is vowel?

**Review by akimovaa — APPROVED** [review]

File: tasks/51_abbreviated-name/abbreviated-name.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+ * // return 'Verbovyi D. O.'
+ * abbreviated('Verbovyi Dmytro Oleksandrovych')
+ * @param {String} str
+ * @return {String}
+ */
+export function abbreviated(str) {
+  return str
+      .split(' ')
+      .map((value, index) => {
+        if (!index) {
```
Comment (KalachevDev) [inline tasks/51_abbreviated-name/abbreviated-name.js:17]: A bit confusing

**Review by KalachevDev — APPROVED** [review]

**Pull request merged**